scripts/create-docker-compose-production.xsh

```diff
@@ -171,7 +170,7 @@ print("✅ Image rebuild and tagged", color=Color.GREEN)
 # push it
 print(f"Pushing {os.environ['DOCKER_LOGIN']}/{_image_name}:{_tag} ...")
 
-echo @(_docker_psswd) | docker login --username @(_docker_login) --password-stdin
+xonsh ./.vscode/tasks.xsh run @(f"docker-login")
```

Comment:
There is little detail here, as this is a string that is not being formatted with other "Python space" variables we could use it directly:

```xonsh
xonsh ./.vscode/tasks.xsh run docker-login
```